### PR TITLE
remove result

### PR DESCRIPTION
**Pull request by Dhroov7**

### CODE REVIEW

**Comment by championswimmer** [comment]
This pull request **fixes 1 alert** when merging 25747b5cf20f1af5786007b5714fcfcaa4ef4c52 into ac1a5cd686e8dab29e51220d1eca9cfb43d0ca42 - [view on LGTM.com](https://lgtm.com/projects/g/coding-blocks/oneauth/rev/pr-fa02c604b8a64446bcdcf1a57ab59f5b93d5cd88)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: scripts/prune/users_multi_email_zero_verified.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ const {db, models: {
 async function runPrune() {
     try {
         // Only deleting older than 01 May 2018
-        const [users, result] = await db.query(`
+        const users = await db.query(`
```
Comment (samlanning) [inline scripts/prune/users_multi_email_zero_verified.js]: This looks like an incorrect fix to me, `users` will not be the same thing as a result of this:

E:g:
```js
const [a, b] = [1, 2] // a = 1, b = 2
const [a] = [1, 2] // a = 1
const a = [1,2] // a = [1, 2]
```

Comment (championswimmer) [reply scripts/prune/users_multi_email_zero_verified.js]: Shit yes, should be [users] not users

Comment (Dhroov7) [reply scripts/prune/users_multi_email_zero_verified.js]: Will solve this. @championswimmer

**Pull request merged**